sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -346,6 +346,18 @@ def match(actual):
     assert_that(result, matcher())
     pipeline.run()
 
+  def test_to_set(self):
+    pipeline = TestPipeline()
+    the_list = [6, 3, 1, 1, 9, 1, 5, 2, 0, 6]
+    pcoll = pipeline | 'start' >> Create(the_list)
+    result = pcoll | 'to list' >> combine.ToSet()
```

Comment:
Saw this in JIRA and made the exact same implementation before I realised the PR was already out :)

Small comment: there's a tiny typo here that doesn't really matter. Should be 'to set'.